components/closebutton/metadata/closebutton.yml

```diff
@@ -1,4 +1,5 @@
 name: Close button
+SpectrumSiteSlug: https://spectrum.adobe.com/page/cards/
```

Comment:
```suggestion
SpectrumSiteSlug: https://spectrum.adobe.com/page/close-button/
```

Reply:
Fixed! Thanks, that change had been accidentally undone.